src/types/content.ts

```diff
@@ -80,7 +80,7 @@ type ListContent = {
 
 type ClaimContent = DecoratedContent & {
   type: 'Claim',
-  claimType: 'Statement' | 'Theorem' | 'Lemma' | 'Proof' | 'Postulate' | 'Hypothesis' | 'Proposition' | 'Corollary',
+  claimType?: 'Statement' | 'Theorem' | 'Lemma' | 'Proof' | 'Postulate' | 'Hypothesis' | 'Proposition' | 'Corollary',
```

Comment:
No change is needed to the template because `claimType` is not used:

https://github.com/elifesciences/enhanced-preprints-client/blob/master/src/utils/content-to-jsx.tsx#L59-L67